explorer/explorerroutes.go

```diff
@@ -1121,7 +1121,11 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if err != nil {
-		exp.StatusPage(w, "search failed", err.Error()+", you can ", WrongNetworkStatusType)
+		if err.Error() == "Looks like you are searching for an address of type P2PK" {
```

Comment:
When possible, always avoid comparing long string literals against an expected value.  This is very error prone as changing the other message will break this one.  This is why we have codes and enumerations defined at the package level.